ADAL/src/ADAuthenticationContext.m

```diff
@@ -382,6 +421,8 @@ - (void)acquireTokenWithResource:(NSString*)resource
     [request setPromptBehavior:promptBehavior];
     [request setUserIdentifier:userId];
     [request setExtraQueryParameters:queryParams];
+    request.tokenCache = self.tokenCache;
```

Comment:
this is the same code for all acquireTokens. Can we have this somewhere in one place? e.g. when we're creating request?